docs/source/distributing-files.rst

```diff
@@ -114,16 +115,17 @@ When an application launches, the following process occurs:
 5. When the application completes, the staging directory is deleted.
 
 
-Using Conda-Pack to Distribute Python Environments
---------------------------------------------------
+Distribute Python Environments With Conda-Pack
+----------------------------------------------
 
 When deploying Python applications, one needs to figure out how to distribute
-any library dependencies. If Python and the required libraries are already
-installed on every node (option 1 above), you can use the local Python and
-avoid this problem completely. If they aren't, then one way to distribute them
-is to use the `conda package manager <https://conda.io/docs/>`__ to create a
-Python environment, and `conda-pack <https://conda.github.io/conda-pack/>`__ to
-package that environment for distribution.
+any library dependencies. If Python and all required libraries are already
+installed on every node (option 1 above), then you can use the local Python and
+avoid this problem completely. However if these libraries aren't installed by
```

Comment:
`However` needs a comma after it if you start a sentence with it. At that point though this sentence gets a bit unwieldy with additional clauses.

```
Otherwise, one way that users can ...
```

as the case is inferred from the previous sentence.